src/main/java/com/minecolonies/coremod/client/gui/GuiFactory.java

```diff
@@ -1,41 +1,41 @@
 package com.minecolonies.coremod.client.gui;
 
-import java.util.Collections;
-import java.util.Set;
-
 import net.minecraft.client.Minecraft;
 import net.minecraft.client.gui.GuiScreen;
 import net.minecraftforge.fml.client.IModGuiFactory;
 
+import java.util.Collections;
+import java.util.Set;
+
 /**
  * This class is required for Forge to initialize the in-game GUI.
  */
-public class GuiFactory implements IModGuiFactory 
+public class GuiFactory implements IModGuiFactory
 {
     @Override
-    public void initialize(Minecraft minecraftInstance) 
-    { 
+    public void initialize(Minecraft minecraftInstance)
+    {
         // We don't need this method for our purpose.
     }
-    
+
     @Override
-    public Class<? extends GuiScreen> mainConfigGuiClass() 
+    public Class<? extends GuiScreen> mainConfigGuiClass()
     {
         return ConfigGUI.class;
     }
-    
+
     @Override
-    public Set<RuntimeOptionCategoryElement> runtimeGuiCategories() 
+    public Set<RuntimeOptionCategoryElement> runtimeGuiCategories()
     {
         return Collections.emptySet();
     }
-    
+
     /**
      * @deprecated This was never fully implemented and will be removed in the future.
      */
     @Override
     @Deprecated
-    public RuntimeOptionGuiHandler getHandlerFor(RuntimeOptionCategoryElement element) 
+    public RuntimeOptionGuiHandler getHandlerFor(RuntimeOptionCategoryElement element)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1133)